src/scripts/main.js

```diff
@@ -1 +1,10 @@
 'use strict';
+
+const largeImage = document.getElementById('largeImg');
+
+for (const link of document.querySelectorAll('.list-item__link')) {
+  link.addEventListener('click', (e) => {
```

Comment:
please rewrite the code to add only one `eventListener` and find the target by `e.target`